math/dlib/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake  1.1
 
-github.setup        davisking dlib 19.17 v
+github.setup        davisking dlib 54a9a5bbf3267386dd39a82fb792bab1bb60796c
```

Comment:
Is there not a release version you can use? In any event, we don't want the commit tag to be the `version`, so you will need to set to manually to something more meaningful. Perhaps use the date.

Reply:
No, unfortunately - the last release was on December 14, 2019 while the fixes are from January 7, 2020.

I've updated the version to use the date.

Reply:
Using the date will also cause problems when going back to a tagged release in the future, as that would not be seen as an update.

I suggest instead making up a version like 19.17.1, so that when 19.18 or whatever comes out, it will not be a problem uodating.

Reply:
Alternatively, stick with the same version as the last release, and just increase the revision.

Reply:
I've changed the version to 19.19.1